CHANGELOG.md

```
@@ -1,28 +1,42 @@
 # Prowide ISO 20022 - CHANGELOG
 
-#### 9.4.6 - May 2024
-  * (PW-1875) Fixed the `ZuluDateTimeAdapter` to convert the datetime to UTC offset if needed
- 
-#### 9.4.5 - May 2024
+#### 10.1.8 - SNAPSHOT
+  * (GH-118) Added `toJsonV9` and `fromJsonV9` in the `AbstractMX` to handle retro-compatibility with Java 8 DateTime elements in JSON serialization
+
+#### 10.1.7 - August 2024
+  * (PW-1958) Fixed the `DefaultMxMetadataStrategy` NPE issue when the amount values are null
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)